addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
```

Comment:
i didn't know that - i dont think it needs arguments but b/c its a mixin - i always add it - will remove